### PR TITLE
add spin plugin release bot to spin up hub

### PR DESCRIPTION
**Pull request by rajatjindal**

### CODE REVIEW

File: content/api/hub/sample_release_bot_golang.md [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+template = "render_hub_content_body"
+date = "2024-01-15T00:22:56Z"
+content-type = "text/html"
+tags = ["golang", "github", "actions"]
+
+[extra]
+author = "rajatjindal"
+type = "hub_document"
+category = "Sample"
+language = "Golang"
```
Comment (mikkelhegn) [inline content/api/hub/sample_release_bot_golang.md:11]:
```suggestion
language = "Go"
```

**Review by mikkelhegn — CHANGES_REQUESTED** [review]

**Review by mikkelhegn — APPROVED** [review]

**Comment by mikkelhegn** [comment]
LGTM

File: content/api/hub/sample_release_bot_golang.md [hunk 2]
```diff
@@ -0,0 +1,21 @@
+title = "Spin Plugin Release Bot - powered by Spin"
+template = "render_hub_content_body"
+date = "2024-01-15T00:22:56Z"
+content-type = "text/html"
+tags = ["golang", "github", "actions"]
```
Comment (itowlson) [inline content/api/hub/sample_release_bot_golang.md:5]: We use `go` for tags elsewhere.

File: content/api/hub/sample_release_bot_golang.md [hunk 3]
```diff
@@ -0,0 +1,21 @@
...
+author = "rajatjindal"
+type = "hub_document"
+category = "Sample"
+language = "Go"
+created_at = "2024-01-15T00:22:56Z"
+last_updated = "2024-01-15T00:22:56Z"
+spin_version = ">v1.5"
+summary =  "A bot automating releases for Spin Plugins - Powered by Spin"
+url = "https://github.com/rajatjindal/spin-plugin-release-bot"
+keywords = "github, golang, webhook"
```
Comment (itowlson) [inline content/api/hub/sample_release_bot_golang.md:17]: Should these keywords be aligned to the tags?  I am not sure how they are used.

Comment (karthik2804) [reply content/api/hub/sample_release_bot_golang.md:17]: Need not be. 
https://github.com/fermyon/developer/pull/1141#discussion_r1456473759

**Pull request merged**